local-cli/server/server.js

```diff
@@ -12,12 +12,16 @@ const chalk = require('chalk');
 const formatBanner = require('./formatBanner');
 const path = require('path');
 const runServer = require('./runServer');
+const findSymlinksPaths = require('./findSymlinksPaths');
 
 /**
  * Starts the React Native Packager Server.
  */
 function server(argv, config, args) {
-  args.projectRoots = args.projectRoots.concat(args.root);
+  args.projectRoots = args.projectRoots.concat(
+    args.root,
+    findSymlinkPaths(path.resolve(process.cwd(), 'node_modules'))
```

Comment:
Adding to project roots totally breaks `react-native run-android` command.

To reproduce:
```
react-native init TestApp
cd TestApp
react-native run-android
```
Error text:

~/work/react-native/packager ~

ENOENT: no such file or directory, scandir '/Users/bestander/work/react-native/packager/node_modules'

~
Process terminated. Press <enter> to close the window